packages/playwright-test/src/util.ts

```diff
@@ -287,7 +287,10 @@ export async function normalizeAndSaveAttachment(outputPath: string, name: strin
     throw new Error(`Exactly one of "path" and "body" must be specified`);
   if (options.path !== undefined) {
     const hash = calculateSha1(options.path);
-    const dest = path.join(outputPath, 'attachments', hash + path.extname(options.path));
+    let sanitizedNamePrefix = '';
+    if (isString(name))
```

Comment:
We should not ignore non-strings, but rather throw an error.

Reply:
Done.